cypress/component/Tabs.cy.tsx

```diff
@@ -51,7 +51,22 @@ it("tab description should render correctly", () => {
   cy.get(tabPanel).should("exist").and("have.text", "Tab description 3");
 });
 
-// Orientation
-// it("", () => {
+// Orientation check
 
-// })
+it("renders orientation correctly", () => {
+  cy.mount(<TabsExample orientation="vertical" />);
+  cy.get(tabs).should("have.attr", "class").and("to.have.string", "vertical");
```

Comment:
have a look in the dom, you might find some other aria stuff relating to orientation we might want to check.